erpnext/patches/v4_0/reset_permissions_for_masters.py

```diff
@@ -16,5 +16,5 @@ def execute():
 		try:
 			reset_perms(doctype)
 		except:
-			print "Error resetting perms for", doctype
+			print("Error resetting perms for", doctype)
 			raise
```

Comment:
You left this out. `raise` in this form is not python 3 compatible. The python 3 compatible form is also not compatible with python 2. However, six is a dependency so use six.reraise